### PR TITLE
TELCODOCS-1571: Update NVIDIA GPU dashboard

### PR DESCRIPTION
**Pull request by empovit**
Version(s):
4.13+

Issue:

The NVIDIA GPU administration dashboard plugin has been depreacated and replaced with a DCGM Dashboard in OpenShift monitoring.

This PR

* Deletes the NVIDIA GPU administration dashboard page
* Updates the GPU dashboard paragraph of NVIDIA GPU architecture overview 
* Points to the official NVIDIA GPU monitoring dashboard documentation
* Fixes the `{product-title}` placeholder on the overview page

Link to docs preview:

https://64465--docspreview.netlify.app/openshift-enterprise/latest/architecture/nvidia-gpu-architecture-overview

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by empovit** [comment]
@openshift/team-documentation could you please take a look? What else do I need to do to have it merged? Thanks

**Comment by kalexand-rh** [comment]
@empovit, since you're one of the developers for this content, we can skip dev review. It still needs to be approved by QE and then peer reviewed.

@wabouhamad, will you please confirm this change?
@StephenJamesSmith, it looks like you wrote the original content. PTAL.

**Review by wabouhamad — COMMENTED** [review]
/lgtm

**Review by sheriff-rh — APPROVED** [review]
LGTM!

**Comment by michaelryanpeter** [comment]
@johnwilkins or @tradej: I think that this PR needs to go through change management before it can be merged. Is there a TELCO writer who can put this through the docs CM process?

File: modules/nvidia-gpu-features.adoc
Comment (tradej) [inline modules/nvidia-gpu-features.adoc:58]: Nit pick: the unit of temperature is a degree Celsius, so IMO it should say `in degrees Celsius` or `in °C`.

**Comment by tradej** [comment]
> @johnwilkins or @tradej: I think that this PR needs to go through change management before it can be merged. Is there a TELCO writer who can put this through the docs CM process?

@StephenJamesSmith can you have a look? Is this tied to any work that you're doing on NVIDIA now?

File: modules/nvidia-gpu-features.adoc
Comment (empovit) [reply modules/nvidia-gpu-features.adoc]: Fixed

**Comment by StephenJamesSmith** [comment]
@tradej @empovit Created this new PR for TELCO tracking. It looks like all Devs and the QE and the Peer Review have signed off. However, it looks the commits still need to be squashed to 1. After that, just the merge review requet is needed.

**Comment by empovit** [comment]
> @tradej @empovit Created this new PR for TELCO tracking. It looks like all Devs and the QE and the Peer Review have signed off. However, it looks the commits still need to be squashed to 1. After that, just the merge review request is needed.

Done

**Comment by tradej** [comment]
@StephenJamesSmith content is removed here, please do change management for this change, thanks.

**Comment by michaelryanpeter** [comment]
@StephenJamesSmith This has to go through change management before it can be merged. Stakeholders in support and customers need to be notified of this change before it is merged. The process is outlined in the OpenShift docs manual. If you have questions about the process, please reach out to on the OCP docs team channel in Slack.

/remove-label merge-review-needed

**Comment by StephenJamesSmith** [comment]
@fabiendupont @egallen Still need your acks (lgtm) on this. Thanks in advance.

**Comment by StephenJamesSmith** [comment]
@fabiendupont @egallen Still need your acks (lgtm) on this. Thanks in advance.

**Comment by StephenJamesSmith** [comment]
@fabiendupont @egallen Still need your acks (lgtm) on this before I can send a Notice of Change to the wider community.  Please do this at your very earliest convenience.  Thanks in advance.

**Comment by empovit** [comment]
Can we get it merged please? Now that all the stakeholders seem to have approved.

**Comment by StephenJamesSmith** [comment]
@empovit Yes, all stakeholders have approved and a Notice of Change note was sent to all of the relevant mailing lists. I've received only one comment about the Notice. I'll apply a merge request label later today.

File: modules/nvidia-gpu-features.adoc
Comment (abrennan89) [inline modules/nvidia-gpu-features.adoc:58]:
```suggestion
You can install a monitoring dashboard to display GPU usage information on the cluster *Observe* page in the {product-title} web console. GPU utilization information includes the number of available GPUs, power consumption (in watts), temperature (in degrees Celsius), utilization (in percent), and other metrics for each GPU.
```
Literal web console elements like page names should be in bold. Also, per the IBMSG, please use page instead of screen.
We use "page" in other places in the docs that refer to the web console. For example: https://docs.openshift.com/container-platform/4.13/applications/creating_applications/odc-creating-applications-using-developer-perspective.html#odc-creating-sample-applications_odc-creating-applications-using-developer-perspective

image

**Review by abrennan89 — COMMENTED** [review]
One comment before merging.

Also, does this need to be cherrypicked to 4.13 and 4.14? The description just has 4.13 rather than 4.13+

Thanks

File: modules/nvidia-gpu-features.adoc
Comment (empovit) [reply modules/nvidia-gpu-features.adoc:58]: The fix is applicable to 4.13 and on. What do you mean 4.13+? There's no documentation for 4.14 yet.

Comment (empovit) [reply modules/nvidia-gpu-features.adoc:58]: Also, would it be OK if I submit a fix for your comment in a separate PR? This one is a month old, and I don't think we have time for another cycle of reviews, notifications, approvals etc., as the customers keep landing on the wrong page and trying to use the deprecated software.

Comment (StephenJamesSmith) [reply modules/nvidia-gpu-features.adoc:58]: @abrennan89 This change is for 4.13 and all future versions.

Comment (abrennan89) [reply modules/nvidia-gpu-features.adoc]: Thanks folks!

**Review by abrennan89 — APPROVED** [review]

**Pull request merged**
